scripts/longtermeffects.lua

```diff
@@ -189,9 +172,7 @@ local function nextRound_new(nRounds, bTimeChanged)
 		if nCurrent ~= 0 and (nCurrent % 10) == 0 and not bTimeChanged then
 			CalendarManager.adjustMinutes(1);
 			CalendarManager.outputTime();
-
-			local nDateinMinutes = TimeManager.getCurrentDateinMinutes();
-			DB.setValue("calendar.dateinminutes", "number", nDateinMinutes); DB.setValue("calendar.dateinminutesstring", "string", tostring(nDateinMinutes))
+			TimeManager.notifyControlsOfUpdate();
```

Comment:
The writing of the date in minutes to calendar.dateinminutes was only being used to notify the controls on the pane to update... the value was never read.  The bug with that is if the value didn't change, it wouldn't be triggered again.  So I changed this to be an always incrementing value so that it fires every single time.